src/Symfony/Component/Notifier/Bridge/MessageMedia/README.md

```diff
@@ -0,0 +1,24 @@
+MessageMedia Notifier
+=================
+
+Provides [MessageMedia](https://messagemedia.com/) integration for Symfony Notifier.
+
+DSN example
+-----------
+
+```
+MESSAGEMEDIA_DSN=messagemedia://API_KEY:API_SECRET@default?from=FROM
+```
+
+where:
+- `API_KEY` is your API key
+- `API_SECRET` is your API secret
+- `FROM` is your sender (optional)
```

Comment:
Can you explain what they expect? A number or a alpha string? If a number, does it need to start with a + sign? If yes it needs to be url encoded in the DSN